scripts/src/java/org/oppia/android/scripts/ExemptionsList.kt

```diff
@@ -0,0 +1,613 @@
+package org.oppia.android.scripts
+
+class ExemptionsList {
```

Comment:
do we need a class or could we use `object`?

Reply:
Okay!
Changed it to an object